tests/ConditionSQLTest.php

```diff
@@ -48,6 +48,24 @@ public function testBasic()
         $this->assertEquals('Sue', $mm['name']);
     }
 
+    public function testNull()
+    {
+        $a = [
+            'user' => [
+                1 => ['id' => 1, 'name' => 'John', 'gender' => 'M'],
+                2 => ['id' => 2, 'name' => 'Sue', 'gender' => 'F'],
+                3 => ['id' => 3, 'name' => 'Null', 'gender' => null],
+            ], ];
+        $this->setDB($a);
+
+        $m = new Model($this->db, 'user');
+        $m->addFields(['name', 'gender']);
+
+        $mm = clone $m;
+        $mm->addCondition('gender', null)->tryLoadAny();
```

Comment:
Could make sense to use loadAny() here. If the condition has some failure and no matching record is found, an exception will be thrown which will contain more useful information than the information that the assertion failed in the next line.

Reply:
Thanks but I improved the test a bit in another direction